src/AutoRest.CSharp/Output/Models/LowLevelRestClient.cs

```diff
@@ -30,11 +31,10 @@ public LowLevelRestClient(OperationGroup operationGroup, BuildContext<LowLevelOu
             Parameters = _builder.GetOrderedParameters ();
             ClientPrefix = GetClientPrefix(operationGroup.Language.Default.Name, context);
             DefaultName = ClientPrefix + ClientSuffix;
-            Description = "";
         }
 
         public Parameter[] Parameters { get; }
-        public string Description { get; }
+        public string Description => BuilderHelpers.EscapeXmlDescription(CreateDescription(_operationGroup, GetClientPrefix(Declaration.Name, _context)));
```

Comment:
Is there a test that covers this scenario?

Reply:
There is _zero_ test coverage for LLC right now.

It is on the list, but right now we're not sure we've settled on the API I'm generating yet even.